htpasswd_cr.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: cluster
 spec:
   identityProviders:
-  - name: htpasswd_provider 
+  - name: developer only
```

Comment:
I never thought it would be that simple :)

Reply:
does this mean I should only click on this if I'm a crc/openshift developer? Or should I click on it if I want to login as the `developer` user?

Reply:
Yes naming is hard. Only if you want to login with the developer account.

Reply:
`login as 'developer'` or `'developer' login`?

Reply:
There is already the prefix "Log in with". Maybe just say "developer" and nothing else?

Reply:
Sounds good, or `'developer' user`